packages/rs-drive-abci/src/query/mod.rs

```diff
@@ -28,20 +29,22 @@ mod tests {
     use drive::drive::batch::DriveOperation::DataContractOperation;
     use platform_version::version::PlatformVersion;
     use std::borrow::Cow;
+    use std::sync::Arc;
 
-    pub fn setup_platform<'a>() -> (TempPlatform<MockCoreRPCLike>, &'a PlatformVersion) {
+    pub fn setup_platform<'a>() -> (
+        TempPlatform<MockCoreRPCLike>,
+        Arc<PlatformState>,
+        &'a PlatformVersion,
+    ) {
         let platform = TestPlatformBuilder::new()
             .build_with_mock_rpc()
             .set_initial_state_structure();
 
-        let platform_version = platform
-            .platform
-            .state
-            .read()
-            .current_platform_version()
-            .unwrap();
+        let platform_state = platform.platform.state.load_full();
```

Comment:
Please write in a comment why we are doing a load_full instead of a load.

Reply:
Sorry, I'm in rush